gridt/src/app/login/login.page.ts

```diff
@@ -26,9 +26,7 @@ export class LoginPage implements OnDestroy {
   }
 
   private cancelAllSubscriptions() {
-    if (this.loginSubscription) {
-      this.loginSubscription.unsubscribe();
-    }
+    this.subscriptions.map( subscription => subscription.unsubscribe());
```

Comment:
Spacing as mentioned before.

Reply:
Maybe we could make this an interface/class? It is being used rather quite a lot.